asistente_ladm_col/gui/change_detection/dlg_change_detection_settings.py

```diff
@@ -266,29 +279,31 @@ def show_message_clean_workspace(self, message):
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "Yes, remove layers")), QMessageBox.YesRole)
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "No, don't remove")), QMessageBox.NoRole)
         msg.addButton(QPushButton(QCoreApplication.translate("ChangeDetectionSettingsDialog", "Cancel")), QMessageBox.RejectRole)
-        reply = msg.exec_()
+        msg.exec_()
 
-        if reply == QMessageBox.Yes:
+        # Get reply when use custom buttons
+        # https://stackoverflow.com/questions/25101171/what-do-the-different-qmessagebox-roles-mean
+        reply = msg.buttonRole(msg.clickedButton())
```

Comment:
Go back to the 'standard' way we use in this plugin. See https://github.com/AgenciaImplementacion/Asistente-LADM_COL/commit/95b3c536829ac9b51c793d8da398ae8b8e6b136f

Reply:
Adjustment made, thanks